### DISCUSSION TITLE
OSD-settings for more options (background image, text, but maybe even more)

### DISCUSSION THREAD

**Discussion by 3DJupp** [X1Plus/X1Plus#132] [Ideas]
Hey,
right now, there are different ways to control the X1Plus installation.: SSH/FTP and of course, mostly via touchscreen and bambu apps (Handy/Studio)
But wouldn't it be nice to include more settings in the gui?
> please select a background from your SD-Card (x*y pixel, png recommended):
similar for other settings 👍

**Comment by jphannifan** [comment]
We are working on this currently. 

There is a way to display a total of 3 custom images right now, which is detailed under the Personalization part of the wiki, but we'll have more options in the future
